config.md

```diff
@@ -308,15 +308,21 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps.
```

Comment:
nit: Specify explicitly that the key and value are of type `string` and that the specification is not expecting any structure like DNS naming, etc..